cvat/apps/engine/settings.py

```diff
@@ -0,0 +1,73 @@
+import os
```

Comment:
Let's merge the file with media_extractors. Also need to rename _get_mime function to get_mimetype